native_client/ctcdecode/README.mozilla

```diff
@@ -0,0 +1,2 @@
+Originally imported from https://github.com/parlance/ctcdecode, commit 140b45860cec6671fb0bf6dbb675073241c0f9b0
```

Comment:
Any specific step to perform the import ? A one-liner to reproduce would be perfect :)

Reply:
This README is mostly here for licensing reasons, not for updating the import whole-sale, but I can write a script, yeah.

Reply:
Don't bother writing a script if it's just about downloading the tree at this sha, though